addons/event_product/views/event_ticket_views.xml

```diff
@@ -90,15 +90,11 @@
         <field name="model">event.event.ticket</field>
         <field name="inherit_id" ref="event.event_event_ticket_view_kanban_from_event"/>
         <field name="arch" type="xml">
-            <field name="name" position="before">
-                <field name="product_id"/>
-                <field name="price"/>
+            <field name="name" position="after">
+                <field class="fw-bold float-end" name="price"/>
```

Comment:
ms-auto instead of float-end

Reply:
Then i have to make the div `d-flex` instead of `flex-row` in the parent view, by the way why do we prefer ms-auto over float-end?